### DISCUSSION TITLE
difficulty in creating a github private package accessible for users with a key

### DISCUSSION THREAD

**Discussion by pprevankar** [community/community#116492] [Packages]
### Select Topic Area

Show & Tell

### Body

Facing difficulty in creating a github private package which can be accessed by only users with a key

**Comment by grandmas-favorite** [comment]
Hi @pprevankar!

Would you be willing to provide a little more activity about what exactly the difficulty is, perhaps with screenshots/code if applicable?

I think it would help myself and other members understand the issue better.

Cheers!

**Reply by pprevankar** [reply]
Hi @grandmas-favorite,

Am able to create the private github-npm package. Had referred:
https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-npm-registry

Hope, this would help!

Regards,
Pratik
